config/database.yml

```diff
@@ -19,9 +19,8 @@ development: &default
   encoding: unicode
   database: champaign
   pool: 5
-  username: postgres
-  password:
-  host: db
+  username: <%= ENV['PG_USERNAME'] %>
+  host: <%= ENV['PG_HOST'] %>
```

Comment:
For the sake of `circleci`, I had to put `username` and `host` onto `ENV`. @EricBoersma not sure what our setup is on production. I hope this change isn't too much of a problem.